api/tests/Feature/UserTest.php

```diff
@@ -72,89 +72,6 @@ protected function setUp(): void
             ]);
     }
 
-    public function testCreateUserDefaultRoles()
```

Comment:
Why did you remove these tests?  You're undeprecating "create user", right?

Reply:
It doesn't make sense to test an action only used in other tests.

Reply:
That's fair.